app/src/main/java/org/mozilla/fenix/components/toolbar/TabCounter.kt

```diff
@@ -199,34 +197,21 @@ open class TabCounter @JvmOverloads constructor(
             ONE_DIGIT_SIZE_RATIO
         }
 
-        if (newRatio != currentTextRatio) {
```

Comment:
Should we keep this check for doing nothing if the ratios match?

Reply:
I don't think this is necessary since it's not running a complicated calculation and Android is going to eat the draw/resize request if we're setting the same size. 

It was good before when it was updating after a long draw request & was doing complicated calculations.

We can add back if we think it'll speed up perf tho